src/table/cells/selections.jsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  selectionStyle: (props) => props,
+});
 
 export default function withSelections(CellComponent) {
   const HOC = (props) => {
     const { cell, selections } = props;
-    const { selectCell, getCellStyle } = selections;
+    const { style, identifyerClass } = selections.getSelectionStyle(cell);
```

Comment:
This feels like it needs to run quite a lot of code to just get the styling for each render.
Each row has to run the 
`  const hasSelectedElemNumber = !!selected.rows?.find((r) => r.qElemNumber === cell.qElemNumber);` 
for example, which could be pre-generated instead?